Rakefile

```diff
@@ -2,6 +2,10 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'rubocop'
+require 'rubocop/rake_task'
+# require 'rubocop-rake'
```

Comment:
If we are not using this line of code we must remove it

Reply:
Agreed :smile: